docs-content/installing.html.md.erb

```diff
@@ -1,34 +1,76 @@
 ---
-title: Installing and Configuring YOUR-PRODUCT-NAME
+title: Installing and Configuring Splunk OpenTelemetry Collector for VMware Tanzu
 owner: Partners
 ---
 
 <strong><%= modified_date %></strong>
 
-This topic describes how to install and configure YOUR-PRODUCT-NAME.
+This topic describes how to install and configure Splunk OpenTelemetry Collector for VMware Tanzu.
 
-##<a id='install'></a> Install and Configure YOUR-PRODUCT-NAME
+##<a id='install'></a> Install and Configure Splunk OpenTelemetry Collector for VMware Tanzu
 
-To install the YOUR-PRODUCT-NAME file on the Ops Manager Installation Dashboard, do the following:
+To install the Splunk OpenTelemetry Collector for VMware Tanzu file on the Ops Manager Installation Dashboard,
+do the following:
 
-1. Download the product file from Pivotal Network.
+1. Download the **Splunk OpenTelemetry Collector for VMware Tanzu** product file from Pivotal Network.
 
 1. Navigate to the Ops Manager Installation Dashboard and click **Import a Product** to upload the product file.
 
-1. Under **Import a Product**, click **+** next to the version number of YOUR-PRODUCT-NAME.
-This adds the tile to your staging area.
+1. Under **Import a Product**, click **+** next to the version number of Splunk OpenTelemetry Collector
+for VMware Tanzu. This adds the tile to your staging area.
 
-1. Click the newly added **YOUR-PRODUCT-NAME** tile.
+1. Click the newly added **Splunk OpenTelemetry Collector for VMware Tanzu** tile to access the
+configuration settings.
 
-1. **This portion of the procedure should tell the users how to configure the newly added tile by
-walking them through the sections. Which fields should they fill out? What are the different
-checkboxes? Include relevant screenshots here.**
+    ![Splunk OpenTelemetry Collector for VMware Tanzu](tanzu_tile_in_ops_mgr.png)
 
-1. Click **Save**.
+1. **Assign AZs and Networks**
 
-1. Return to the Ops Manager Installation Dashboard and click **Apply changes** to install YOUR-PRODUCT-NAME tile.
+    1. Choose placement.
 
-**If needed, include additional configuration information below. For example, maybe there are
-configuration options not required for installation but which could be useful for certain users. Typical topics include:**
-1. **Self-service**--Do you have recommendations on how to configure/design plans in the tile so that users can choose the best options for themselves?
-1. **High availability**--How do you configure/deploy X Service tile to be highly available?
+    1. Click **Save**.
+
+1. **Nozzle Config**
+
+    ![Splunk OpenTelemetry Collector nozzle configuration](tanzu_nozzle_config.png)
+
+    1. **RLP gateway shard ID**: Metrics are load balanced between receivers that use the same shard ID.
+    Only change from the default value if multiple receivers must receive all metrics instead of balancing metrics
```

Comment:
"Only modify the default value"

Reply:
Is it "multiple receivers must receive all metrics" or "all receivers must receive all metrics"?

Reply:
It's multiple receivers must receive all metrics. 

For example, let's say a user is running 3 collectors with cloud foundry receivers, all pointing to the same RLP gateway. One has a shard ID `opentelemetry`, and two have `non-default-shard-id`. All metrics will be sent to the group of receivers with `opentelemetry` shard ID, and all will be sent to the group of receivers with `non-default-shard-id` shard ID. So the `opentelemetry` receiver will get all metrics, and then each of the `non-default-shard-id` receivers will get roughly half, since the metrics are balanced across the group of `non-default-shard-id` receivers.

Reply:
I've updated the wording of "only modify the default value" as requested, but let me know if I should make other changes as well.

Reply:
Thanks! LGTM now!